exercises/two-fer/README.md

```diff
@@ -0,0 +1,25 @@
+# Two Fer
+
+Create a sentence of the form "One for X, one for me."
+
+`Two-fer` or `2-fer` is short for two for one. One for you and one for me.
+
+```plain
+"One for X, one for me."
+```
+
+When X is a name or "you".
+
+If the given name is "Alice", the result should be "One for Alice, one for me." If no name is given, the result should be "One for you, one for me."
+
+### Submitting Exercises
+
+Note that, when trying to submit an exercise, make sure the exercise file that you're submitting is in the `exercism/csharp/<exerciseName>` directory.
+
+For example, if you're submitting `bob.cs` for the Bob exercise, the submit command would be something like `exercism submit <path_to_exercism_dir>/csharp/bob/bob.cs`.
+
+## Source
+This is an exercise to introduce users to basic programming constructs, just after Hello World. [https://en.wikipedia.org/wiki/Two-fer](https://en.wikipedia.org/wiki/Two-fer)
+
+## Submitting Incomplete Solutions
+It's possible to submit an incomplete solution so you can see how others have completed the exercise.
```

Comment:
Should we be adding a hints to documentation on optional values? E.g. [this page](https://fsharpforfunandprofit.com/posts/the-option-type/).